developers/bindings/thing-xml.md

```diff
@@ -220,8 +220,13 @@ The following snippet shows a 'Control' and 'Light' tags definition:
 </tags>
 ```
 
-Please note that only [Point](https://openhab.org/javadoc/latest/org/openhab/core/semantics/model/point/package-summary.html) and/or [Property](https://openhab.org/javadoc/latest/org/openhab/core/semantics/model/property/package-summary.html) tags from the pre-defined tag library should be used.
-See also the [Semantic Model]({{base}}/tutorial/model.html) for more information on how tags are used with items.
+There are two possible types of tag -- namely "Semantic" and "Non Semantic" tags.
```

Comment:
IMO, these should only be used to add "Semantic" tags. We don't want bindings to start injecting non semantic tags too, do we?

Reply:
I dunno. That seems to have been the opinion of some others on the community. But I would happily go for Semantic ones only. (Maybe say that if a developer finds a new semantic tag that they want added, then they should include a PR towards OH core for that.

Reply:
> IMO, these should only be used to add "Semantic" tags. We don't want bindings to start injecting non semantic tags too, do we?

THERFORE we can (and should) include only the allowed point and property tags in the XSD schema. This will indeed prevent developers from adding other stuff, and force a degree of quality control.

NB and we should make a note that whenever the .csv file is updated the XSD should be updated too.